koku/masu/test/processor/ocp/test_ocp_cloud_report_summary_updater.py

```diff
@@ -255,52 +257,55 @@ def test_get_infra_map_from_providers(self):
     # @patch(
     #     "masu.processor.ocp.ocp_cloud_parquet_summary_updater.OCPReportDBAccessor.populate_ocp_on_all_project_daily_summary"  # noqa: E501
     # )
-    @patch("masu.database.cost_model_db_accessor.CostModelDBAccessor.cost_model")
-    def test_update_summary_tables_azure(
-        self, mock_cost_model  # , mock_ocpall_proj_summ, mock_ocpall_summ, mock_ocpall_persp
-    ):
-        """Test that summary tables are updated correctly."""
-        markup = {"value": 10, "unit": "percent"}
-        mock_cost_model.markup = markup
-
-        start_date = self.dh.this_month_start
-        end_date = self.dh.this_month_end
-
-        updater = OCPCloudReportSummaryUpdater(schema=self.schema, provider=self.azure_provider, manifest=None)
-
-        updater.update_summary_tables(
-            start_date,
-            end_date,
-            self.ocp_on_azure_ocp_provider.uuid,
-            self.azure_provider.uuid,
-            Provider.PROVIDER_AZURE,
-        )
-
-        summary_table_name = AZURE_REPORT_TABLE_MAP["ocp_on_azure_daily_summary"]
-        with AzureReportDBAccessor(self.schema) as azure_accessor:
-            query = azure_accessor._get_db_obj_query(summary_table_name).filter(
-                cost_entry_bill__billing_period_start=start_date
-            )
-            markup_cost = query.aggregate(Sum("markup_cost"))["markup_cost__sum"]
-            pretax_cost = query.aggregate(Sum("pretax_cost"))["pretax_cost__sum"]
-
-        self.assertAlmostEqual(markup_cost, pretax_cost * decimal.Decimal(markup.get("value") / 100), places=5)
-
-        daily_summary_table_name = OCP_REPORT_TABLE_MAP["line_item_daily_summary"]
-        with OCPReportDBAccessor(self.schema) as ocp_accessor:
-            query = ocp_accessor._get_db_obj_query(daily_summary_table_name).filter(
-                report_period__provider=self.ocp_on_azure_ocp_provider,
-                report_period__report_period_start=self.dh.this_month_start,
-            )
-            infra_cost = query.aggregate(Sum("infrastructure_raw_cost"))["infrastructure_raw_cost__sum"]
-            project_infra_cost = query.aggregate(Sum("infrastructure_project_raw_cost"))[
-                "infrastructure_project_raw_cost__sum"
-            ]
-
-        self.assertIsNotNone(infra_cost)
-        self.assertIsNotNone(project_infra_cost)
-        self.assertNotEqual(infra_cost, decimal.Decimal(0))
-        self.assertNotEqual(project_infra_cost, decimal.Decimal(0))
+    # @patch("masu.database.cost_model_db_accessor.CostModelDBAccessor.cost_model")
```

Comment:
For this and the other commented out tests, would it be better to just skip them them with a reason? That way we'll still see it in test output and be reminded to remove/revisit it eventually. We could also make the skips conditional using an env var to make it easy to run skipped tests without having to change the code.

```suggestion
@unittest.skipIf(os.getenv("RUN_SKIPPED_TESTS"), "Revisit this test")
@patch("masu.database.cost_model_db_accessor.CostModelDBAccessor.cost_model")
```

Reply:
thanks Sam. I add a skip on some tests that may require revisiting